core-features/generate-client-and-cli-install-npm-lockfile/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+if [ -f "pnpm-lock.yaml" ] || [ -f "yarn.lock" ]; then
+  echo "Test should have produced a package-lock.json file only"
+  exit 1
+fi
```

Comment:
before running test?

Reply:
Yes, the test check the contents of the package.json and this looks that only the correct lockfile was created by generate.